docs/reference/remap/functions/parse_key_value.cue

```diff
@@ -46,6 +46,13 @@ remap: functions: parse_key_value: {
 			default: "lenient"
 			type: ["string"]
 		},
+		{
+			name:        "accept_standalone_key"
```

Comment:
This seems _verbose_ but I can't think of anything more concise that's still self-explanatory